### DISCUSSION TITLE
Is there a commercialized version of this that I can buy off the shelf?

### DISCUSSION THREAD

**Discussion by nickkos101** [HumanAIGC/OutfitAnyone#25] [Q&A]
I run a series of e-commerce fashion related stores and I was wondering if there is a commercialized version of this out there somewhere that I could utilize? Hopefully one that lets me run large batches of different garments and model types.

**Comment by lgiavedoni** [comment]
hi @nickkos101, any luck finding it? I am also interested on it for my store. Any pointers will be greatly appreciated

**Comment by VolodimirShutov** [comment]
Hi.
Great job. I'm very interested in your plugin.
I represent companies that create games. Your plugin can help us a lot in creating characters for games. Is it possible to contact you and discuss how we could use it?
How the plugin will behave with high-quality resolutions
Thank you.

**Comment by Vananh157** [comment]
Hello!!! I am managing a fashion website. I am interested in integrating your AI onto the website. I am very pleased to collaborate with you.

**Comment by sevnightingale** [comment]
Hey @nickkos101 @lgiavedoni @VolodimirShutov @Vananh157 !

I'm currently working on a tool that can do this, like adding your apparel items on custom virtual models. Sounds like you might be looking for something similar?

Would love to understand how you're trying to use these tools to do and what features you'd like to see. Here's my calendly if any of of you would be willing to have a quick chat: https://calendly.com/sev-nightingale/15-min-chat

**Comment by aadinash** [comment]
@nickkos101 @lgiavedoni @VolodimirShutov @Vananh157 
We're also creating something similar! I've attached some results with celebrities and different body types, etc. Our approach is very different from most people, but it has a few advantages including
1. You can change the pose and background easily while maintaining body accuracy
2. Users don't have to upload a full-body photo, just a selfie and some information about their body-type
3. Great consistency (no copy-paste feel, no flares or artifacts) and face doesn't get messed up

Let's get in touch. Contact me at anashikkar0 AT gmail.com or, I guess, I'll also drop my calendly :) https://calendly.com/aadinashikkar/30min
<img width="1677" alt="demoPics" src="https://github.com/HumanAIGC/OutfitAnyone/assets/66572810/b8ac69da-1c48-464b-9f33-79db9a97d4e7">
